dm/tests/openapi/run.sh

```diff
@@ -228,6 +228,72 @@ function test_dump_task() {
 
 }
 
+function test_load_task() {
+	echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>START TEST OPENAPI: dump & load TASK"
+	prepare_database
+
+	task_name_dump="test-dump"
+	task_name_load="test-load"
+
+	# create source successfully
+	openapi_source_check "create_source1_success"
+	# get source list success
+	openapi_source_check "list_source_success" 1
+
+	# create source successfully
+	openapi_source_check "create_source2_success"
+	# get source list success
+	openapi_source_check "list_source_success" 2
+
+	# get source status success
+	openapi_source_check "get_source_status_success" "mysql-01"
+
+	# create task success: not valid task create request
+	openapi_task_check "create_task_failed"
+
+	# create dump task success
+	openapi_task_check "create_dump_task_success"
+	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"query-status $task_name_dump" \
+		"\"stage\": \"Stopped\"" 1
+
+	init_dump_data
```

Comment:
in https://github.com/pingcap/tiflow/pull/11729, `function test_dump_task() {` also didn't check whether the data is correct, can you fix it in this pr too?

Reply:
On a second thought, the `test_dump_task()` is kind of duplicate since dump and load task already cover the case. Thus we don't really need a standalone dump test. Dump task was removed and I renamed `test_load_task` to clarify that. Load test itself uses sync diff to check the results, as mention above. Does this work for you?